### ISSUE TITLE
[Atlas] Modifier le fond de carte

### ISSUE THREAD

**Issue by jpm-cbna** [cbn-alpin/sinp-paca-tickets#80]
Dans la _**fiche commune**_ et dans la _**fiche taxon**_, y a-t-il possibilité de modifier le fond de carte et en particulier d'ajouter des zonages ?

_Origine_: Antoine R.

**Comment by jpm-cbna** [comment]
Oui, il est possible de modifier le fond de carte et d'en proposer plusieurs.
Il faudrait m'indiquer celui qui est souhaité.

Par contre, il ne me semble pas possible par défaut avec l'Atlas d'ajouter des couches supplémentaires au fond cartographique.

**Comment by okristo** [comment]
Lorsqu'on clique sur Earth (en bas à gauche de la carte), on passe d'OSM à IGN mais pas aux photos aériennes comme attendu. A modifier.

**Comment by jpm-cbna** [comment]
Corrigé. C'était un problème de configuration. Nous avions utilisé la carte Topo OSM durant la période de transition des web services du GeoPortail vers la GeoPlateforme de l'IGN...

J'ai également bloqué la carte pour ne pas pouvoir dé-zoomer trop.
